lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -49,6 +49,17 @@ def on_send(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            expectation = node.parent.parent
+            eq = expectation.children.last.children.last
+            return unless eq.is_a?(RuboCop::AST::Node)
```

Comment:
What is this case when it doesn't auto-correct? Do you mind to add an example or extend an existing one with `expect_no_correction` to cover it?

Reply:
Hi @pirj, great question. This guard clause is for an entry in `spec/smoke_tests/weird_rspec_spec.rb`:
```
expect([]).to be_empty
```
In this case, there wasn't an obvious way to autocorrect it (although I'm new to rspec), so it would still need to be manually refactored.

Reply:
It seems that what you are doing here is basically checking whether the mather has arguments.

What will happen with `expect([1,2,3]).to include(a)`?

Perhaps this should not be flagged in the first place, but the correction also doesn't make sense. I guess you should check the actual matcher, and whether it has arguments or not.

Reply:
Maybe add a similar non-auto-correctable example to ExpectActual's spec?

Reply:
Great catch, i've scoped the autocorrect to only apply when the right-hand side is an `eq`. I've added a test using @Darhazer's example.

Please take another look. Thanks!